### DISCUSSION TITLE
Redirect URL

### DISCUSSION THREAD

**Discussion by OliverTeo288** [polymind-inc/acmebot#521] [Q&A] [answered]
Is it possible to edit the redirect url of the azure oauth2 authentication? 

Currently I banished the function app to a private endpoint and only accessible through an application gateway with a custom domain as well. However, once i tried logging in with Azure AD, it redirects me to the function app domain instead of the custom domain

**Comment by shibayan** [comment, marked as answer]
Read the following document and change the forwardProxy setting from NoProxy to Standard.
https://azure.github.io/AppService/2021/03/26/Secure-resilient-site-with-custom-domain.html

**Reply by OliverTeo288** [reply]
Thanks alot @shibayan helps alot !
